Small tweaks to handlebars-basics.md (#571)
ember-learn/guides-source: guides/release/templates/handlebars-basics.md

</p>
```

> When you start an app with `ember serve`, your templates are compiled down to something that Embers rendering engine can process more easily. The compiler helps you catch some errors, such as forgetting to close a tag or missing a quotation mark.

Reviewer: ```suggestion
When you start an app with `ember serve`, your templates are compiled down to something that Ember's rendering engine can process more easily. The compiler helps you catch some errors, such as forgetting to close a tag or missing a quotation mark.
```

Ember Helpers are a way to use JavaScript logic in your templates.
For example, you could write a Helper function that capitalizes a word, does some math, converts a currency, or more.

> A Helper takes in two types of arguments `positional` (an array of the positional values passed in the template) and `named` (an object of the named values passed in the template) that are passed into the function, and should return a value.

Reviewer: ```suggestion
A Helper takes in two types of arguments `positional` (an array of the positional values passed in the template) or `named` (an object of the named values passed in the template), which are passed into the function, and should return a value.
```

Ember ships with several built-in helpers, which you will learn more about in the following guides.

> ### Nested Helper Invocations

Reviewer 2: Should this be `Statements` instead of `Invocations`? 

I'm worried about `invocation` sounding a bit scary to newcomers.

Reviewer: ```suggestion
### Nested Helpers
```

I would like to keep this as "nested helpers" since it is a pre-existing anchor tag. Whenever possible, we try to keep our URLs alive, including anchors. Exceptions are when the topic naming is outdated.

Author: Seems good, my concern is _mostly_ that folks will think of "nested helpers" in the same way that they do "nested components" (which usually means "components in subdirectories").
